src/features/filters/components/filter-builder.tsx

```diff
@@ -292,22 +292,23 @@ function FilterBuilderForm({
                   ) : filter.type === "number" ||
                     filter.type === "numberObject" ? (
                     <Input
-                      value={filter.value?.toString() ?? ""}
+                      value={filter.value?.toString().replaceAll(",", ".") ?? ""}
```

Comment:
Were you able to reproduce the issue, that we had? With this implementation, i still have the `,` in the input field. Also, i am not able to type a point into the field myself.

Reply:
On our end it's the opposite, we tested after changing the language settings in the browser and were only able to input `.` . We'll take a deeper look into this matter.